M2/Macaulay2/m2/packages.m2

```diff
@@ -581,6 +581,12 @@ debug GlobalDictionary := dict -> (
     if not isMember(dict, dictionaryPath) then dictionaryPath = prepend(dict, dictionaryPath);
     checkShadow())
 
+locate Package := pkg -> NumberedVerticalList nonnull (
```

Comment:
I'm a little torn about the return value here.  On one hand, it seems that `locate` should return a single `FilePosition` object, but on the other hand, it does make sense to list all the files.

Is there a reason for `NumberedVerticalList`?  For things like `methods` and `about` where there's a `ZZ` method we can run right afterwards, that makes sense, but I"m wondering if just `VerticalList` would suffice?

Reply:
It's not in its final form, hence why it's not really mentioned in the documentation, but I wanted to put it out there in case others want to shape it differently.

There are very few occasions when it makes sense to use VerticalList but not NumberedVerticalList, and those are when you explicitly don't want additional numbers listed on the left. This wasn't a case like that.